Orange/widgets/model/icons/NN.svg

```diff
@@ -0,0 +1,48 @@
+	l3.197-1.014l3.962,4.189l-1.01,1.247l4.153,0.445l-1.3-3.969l-1.209,1.492l-3.539-3.741l1.739-0.552l0.458,1.728l2.811-3.091
+	l-3.971-1.292l0.447,1.688l-2.247,0.712l-2.189-2.315l2.819-3.094l2.076,0.657l-0.478,1.806l3.971-1.292l-2.811-3.091l-0.426,1.61
+	l-1.59-0.503l2.736-3.003l1.141,1.408l1.299-3.969l-4.153,0.445l1.077,1.329l-3.15,3.457l-3.522-1.115
+	c0.174-1.242-0.242-2.434-1.047-3.29l6.487-2.428l0.279,1.821l3.123-2.772l-3.811-1.711l0.253,1.652l-7.255,2.715
+	c-0.383-0.219-0.799-0.392-1.257-0.476c-2.173-0.398-4.257,1.039-4.653,3.211c-0.399,2.176,1.041,4.261,3.212,4.656
+	c0.775,0.143,1.535,0.042,2.214-0.232l2.101,2.221l-2.146,2.355c-0.232-0.091-0.474-0.164-0.728-0.211
+	c-2.173-0.398-4.257,1.039-4.653,3.211c-0.399,2.176,1.041,4.261,3.212,4.656c1.079,0.198,2.134-0.059,2.974-0.631l6.947,2.716
+	l-0.437,1.708l3.961-1.328l-2.837-3.064L16.916,35.077z M9.856,21.229c0.567-0.424,1.027-0.991,1.307-1.67l3.032,0.96l-2.458,2.698
+	L9.856,21.229z M11.75,24.687l1.806,1.909l-2.984,0.947c-0.22-0.274-0.473-0.521-0.76-0.73L11.75,24.687z"/>
+<g>
+	<path fill="#989898" d="M39.745,27.969c2.173,0.399,4.259-1.04,4.655-3.214c0.396-2.173-1.043-4.256-3.214-4.653
+		c-2.173-0.398-4.257,1.039-4.653,3.211C36.134,25.488,37.574,27.573,39.745,27.969z"/>
+</g>
+<g>
+	<path fill="#989898" d="M39.812,15.969c2.173,0.399,4.259-1.04,4.655-3.214c0.396-2.173-1.043-4.256-3.214-4.653
+		c-2.173-0.398-4.257,1.039-4.653,3.211C36.2,13.488,37.641,15.573,39.812,15.969z"/>
+</g>
+<g>
+	<path fill="#989898" d="M39.812,39.969c2.173,0.399,4.259-1.04,4.655-3.214c0.396-2.173-1.043-4.256-3.214-4.653
+		c-2.173-0.398-4.257,1.039-4.653,3.211C36.2,37.488,37.641,39.573,39.812,39.969z"/>
+</g>
+<path fill="#666666" d="M32.628,38.258l-4.854,0.019c0.303-0.446,0.523-0.957,0.626-1.521c0.171-0.936-0.015-1.845-0.436-2.619
+	l6.171-4.706l0.7,0.958l-1.037,0.904l4.013,1.155l-0.593-4.135l-1.054,0.918l-0.187-0.3l0.734-1.919l-1.859,0.107l-2.396-3.857
+	l1.182-1.649l3.15-0.09l-1.245-2.568l0.319-0.445l1.432,1.193l0.503-4.146L33.812,16.8l1.281,1.067l-0.045,0.063l-0.081-0.167
+	l-0.819,1.352l-5.911-5.792c0.063-0.185,0.128-0.37,0.164-0.568c0.167-0.916-0.007-1.808-0.41-2.572l4.481-0.017l-0.308,1.201
+	l3.961-1.323l-2.834-3.07l-0.562,2.191l-5.458,0.021c-0.552-0.544-1.266-0.934-2.084-1.084c-2.172-0.398-4.256,1.039-4.653,3.211
+	c-0.399,2.176,1.041,4.261,3.212,4.656c0.969,0.178,1.913-0.025,2.705-0.481l4.805,7.734l-6.347,8.857
+	c-2.068-0.231-3.995,1.155-4.374,3.233c-0.399,2.176,1.041,4.261,3.212,4.656c1.112,0.204,2.197-0.078,3.048-0.688l5.882-0.022
+	l0.084,1.77l3.396-2.431l-3.612-2.101L32.628,38.258z M27.257,14.891c0.192-0.188,0.368-0.393,0.521-0.619l5.812,5.694l-1.703,2.376
+	L27.257,14.891z M33.717,27.185l-1.173,0.067l1.002,1.371l-6.161,4.698c-0.393-0.423-0.869-0.772-1.424-0.996l5.865-8.184
+	L33.717,27.185z"/>
+<path fill="#989898" d="M28.403,24.722l4.067-0.016l0.084,1.769l3.396-2.431l-3.612-2.101l0.084,1.762l-3.985,0.016
+	C28.464,24.05,28.462,24.383,28.403,24.722z"/>
+</svg>

```

Comment:
Icon is kind of crowded, lines are too thin... Maybe remove some arrows, and reduce the right-most layer to two neurons.

Reply:
Icon is taken from Orange2 and will be replaced with something nicer from Lara. Soon. :)